src/app/image/image.component.html

```diff
@@ -1 +1 @@
-<img [src]="displayedImage">
+<img [src]="buildImage(src)">
```

Comment:
what does `buildImage` do? how do you build an image? is there a better name for this method?